modules/virt-defining-storageclass-in-cdi.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-preparing-cdi-scratch-space.adoc
+
+[id="virt-defining-storageclass-in-cdi_{context}"]
+= Defining a storage class in the CDI object
+
+Define a storage class in the `CDI` object  to dynamically provision scratch space for CDI operations.
+
+.Procedure
+
+* Use the `oc` client to edit the `CDI` object and edit the value of the `spec.config.scratchSpaceStorageClass` field to match a storage class in the cluster.
```

Comment:
I would prefer it if this were actually two steps (step 1: open the CDI object for editing by running the following command; step 2: edit the `spec.config.scratchSpaceStorageClass` field). I know this is debatable, though.

Reply:
Works for me. 
I also added a third step to 'save and exit to update'. Gone from a one-step to a three-stepper with a prereq!